### DISCUSSION TITLE
Add WiX insignia support

### DISCUSSION THREAD

**Discussion by patriksvensson** [cake-build/cake#3347] [Extension requests]

**Comment by gep13** [comment]
@patriksvensson what is WiX Insignia? #lazyweb

**Comment by patriksvensson** [comment]
@gep13 It's a tool used to detach and attach the burn engine from a bundle to be able to properly sign it. http://wixtoolset.org/documentation/manual/v3/overview/insignia.html

**Comment by comradelemoncake** [comment]
I'd love to help with this!

**Comment by augustoproiete** [comment]
Related docs: https://wixtoolset.org/documentation/manual/v3/overview/insignia.html
